crates/python/src/executable.rs

```diff
@@ -31,13 +34,29 @@ py_wrap_type! {
 
 impl_as_mut_for_wrapper!(PyExecutable);
 
-#[pyclass]
-#[pyo3(name = "ExeParameter")]
-#[derive(FromPyObject)]
-pub struct PyParameter {
-    name: String,
-    index: usize,
-    value: f64,
+#[derive(Clone, Debug)]
+pub struct ExeParameter {
```

Comment:
The `#[pyclass]` wasn't playing well with `wrap_data_struct` for getters/settings and `#[pymethods]` for the constructor - this seems to work though.

Reply:
Yeah, `py_wrap_data_struct` creates a new `pyclass`

Reply:
It didn't generate the getters or setters. Possibly because the fields were not public though...

Reply:
When I just have
```rs
#[pyclass]
#[pyo3(name = "ExeParameter")]
#[derive(FromPyObject)]
pub struct PyParameter {
    pub name: String,
    pub index: usize,
    pub value: f64,
}
```

I get the following from the test:
```
tests/test_api.py::test_exe_parameters - TypeError: No constructor defined
```

Then adding the following
```rs
#[pymethods]
impl PyParameter {
    #[new]
    pub fn new(name: String, index: usize, value: f64) -> Self {
        Self { name, index, value }
    }
}
```

I get this from the test
```py
    def test_exe_parameters():
        """Should be able to construct and pass exe parameters"""
        exe_parameter = qcs_sdk.ExeParameter("a", 1, 2.5)
>       assert exe_parameter.name == "a"
E       AttributeError: 'builtins.ExeParameter' object has no attribute 'name'
```

I figured the data struct would be the easiest way to add getters/setters, unless there is a recommended way with `#[pyclass]`

Reply:
I was thinking it would be an automatic thing, but looks like it works [if you annotate each field](https://pyo3.rs/v0.17.0/class#object-properties-using-pyo3get-set) with `#[pyo3(get, set)]`

Reply:
.... man I really need to learn to RTFM. Thank you, fixed...